src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Lotto {
+
+    private final List<Integer> lottoNumbers;
+    public static final int MIN_LOTTO_NUMBER = 1;
+    public static final int MAX_LOTTO_NUMBER = 45;
+    private static final int SUBLIST_START_IDX = 0;
+    private static final int SUBLIST_END_IDX = 6;
+
+    public Lotto(){
```

Comment:
```suggestion
    public Lotto() {
```

자동 포멧팅세팅 해주세요 :pray: